targets/libjpeg-turbo/libjpeg_turbo_fuzzer.cc

```diff
@@ -0,0 +1,28 @@
+#include <stdint.h>
+#include <stdlib.h>
+
+#include <turbojpeg.h>
+
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+    tjhandle jpegDecompressor = tjInitDecompress();
+
+    int width, height, subsamp, colorspace;
+    int res = tjDecompressHeader3(
+        jpegDecompressor, data, size, &width, &height, &subsamp, &colorspace);
+
+    if (res != 0 || width == 0 || height == 0) {
```

Comment:
We will need some protection against zip-bombs. E.g. check for width * height < 1M